ConsumerEdition/guides/zram_swapspace.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Create Swap Space using ZRAM or RAM Compression
+---
+# Create Swap Space using ZRAM or RAM Compression
+
+ZRAM enable us to create extra swap space but without the cons of slower speeds due to Hard Drive or SD Card limitations.
+It is not as fast as actual RAM, but it is close enough for the speed degradation to not be noticable.
+
+## Hardware
+
+- 96Boards Consumer Edition board (tested with DragonBoard410c)
+
+## Software
+
+- Linaro Debian Operating system snapshot Builds (http://builds.96boards.org/snapshots/)
+- If you are not using the DragonBoard410c, you will need to rebuild the kernel using ```CONFIG_ZRAM=m``` Kconfig option.
```

Comment:
i had to do this in our defconfig to enable ZRAM:

+CONFIG_ZSMALLOC=m
+CONFIG_ZRAM=m

Reply:
Doh! yes ofc. I forgot about that.